src/main/java/com/google/gerrit/extensions/common/Approvals.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2014 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+package com.google.gerrit.extensions.common;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Approvals {
+
+  @SerializedName("Code-Review")
+  public String codeReview;
```

Comment:
Labels are not "hardcoded" in Gerrit - any Gerrit instance can define its own labels (Code-Review and Verified are just default labels). So you should use a `Map<String, Integer> approvals` in `ReviewerInfo` instead. This class can be removed.